src/scripts/AirdropTxBuilder.ts

```diff
@@ -0,0 +1,104 @@
+import { TxBuilderModel } from '../model/TxBuilderModel';
+import { norm } from '../utils/TokenUtils';
+import { ReadJSON, WriteJSON } from '../utils/Utils';
+
+const PREGUILD_ADDRESS = '0xe38d06840c9E527b8D40309CCcF4B05af0F888A5';
+const TOKEN_DECIMALS = 18; // TODO MAKE IT DYNAMIC ? OR PASS AS ARGUMENT ?
```

Comment:
I'd say we can ditch this & remove the < 100 tokens = no airdrop filter at the bottom, but change is not needed to merge

Reply:
Done